CHANGELOG.md

```diff
@@ -6,12 +6,12 @@ Security Update
 
 ### Notable changes
 
-* **http**: Fix a bug where an HTTP socket may no longer have a socket but a pipelined request triggers a pause or resume, a potential denial-of-service vector. (Fedor Indutny)
+* **http**: Fix CVE-2015-8027, a bug whereby an HTTP socket may no longer have a parser associated with it but a pipelined request attempts trigger a pause or resume on the non-existent parser, a potential denial-of-service vulnerability. (Fedor Indutny)
```

Comment:
I speculate that `attempts trigger` -> `attempts to trigger`
(sorry If I'm wrong)